src/rgw/CMakeLists.txt

```diff
@@ -189,6 +190,10 @@ add_dependencies(radosgw cls_rgw cls_lock cls_refcount
   cls_version cls_replica_log cls_user)
 install(TARGETS radosgw DESTINATION bin)
 
+if (WITH_CURL_OPENSSL)
+  target_link_libraries(radosgw ${OPENSSL_LIBRARIES})
```

Comment:
@cbodley I assume we should be using `radosgw_a` at this point?

Reply:
several `radosgw-admin` commands use curl too, so `rgw_a` might be the right target.. although we aren't doing any of the curl global init stuff in rgw_admin.cc - do you think we should?

Reply:
so we use the HTTP manager which calls a curl_multi_init; so maybe that does the requisite setup?